creator.html

```diff
@@ -10,21 +10,16 @@
                 <div class="mask rgba-black-light d-flex justify-content-center align-items-center">
                     <div class="text-center white-text mx-5 wow fadeIn">
                         <h1 class="mb-4">
-                            <strong>Convert your ideas into apps with Kodular!</strong>
+                            <strong>Convert Your Ideas Into Apps With Kodular!</strong>
                         </h1>
                         <p>
-                            <strong>The best online drag-and-drop app creator</strong>
-                        </p>
-                        <p class="mb-4 d-none d-md-block">
-                            <strong>The most complete creator for Android apps is here, and it's FREE!<br>
-                                Build amazing projects without any coding knowledge with Kodular, just drag some components and join a few blocks!
-                            </strong>
+                            <strong>The best online drag n' drop app creator without coding.</strong>
                         </p>
                         <a class="btn btn-lg btn-rounded deep-purple accent-4 text-uppercase" target="_blank" href="https://creator.kodular.io">
-                            <i class="fa fa-cloud left"></i> Create Apps!
+                            <i class="fa fa-cloud left"></i> GET STARTED-FREE FOREVER
```

Comment:
```suggestion
                            <i class="fa fa-cloud left"></i> Get Started
```

1. This cannot be with all capitals, and the button has to be smaller. "Get Started" will be enough.